scripts/mounts.env

```diff
@@ -56,7 +58,8 @@ python_mount_list_str="1:3.8.2:/allpython/3.8.2/usr/lpp/IBM/cyp/v3r8/pyz:/allpyt
 "3:3.9:/allpython/3.9/usr/lpp/IBM/cyp/v3r9/pyz:/allpython/3.9:IMSTESTU.PYZ.V39016.ZFS "\
 "4:3.10:/allpython/3.10/usr/lpp/IBM/cyp/v3r10/pyz:/allpython/3.10:IMSTESTU.PYZ.V3A09.ZFS "\
 "5:3.11:/allpython/3.11/usr/lpp/IBM/cyp/v3r11/pyz:/allpython/3.11:IMSTESTU.PYZ.V3B02.ZFS "\
```

Comment:
technically, this (5) has been changed from 3.11 to 3.11-5

Reply:
I'll update the `.ac` and other files as part of the work load balancer work I have for next sprint, I think all the config files for .ac will need refreshing and new ones for ansible-core.2.16 so i can all be handled there.